docs/payid-discovery.md

```diff
@@ -0,0 +1,46 @@
+---
+id: payid-discovery
+title: Discover a PayID
+sidebar_label: Discover a PayID
+---
+
+The PayID Discovery endpoint facilitates finding the PayID URI for a given PayID.
+
+## PayID discovery by a wallet application
+
+Suppose Alice wants to send Bob some XRP from a web-based wallet provider that Alice has an account on. Alice logs in to the wallet provider and enter Bob's PayID, such as `bob$receiver.example.com`, into the wallet UI to start the payment. If the wallet application is set up to use the PayID Discovery endpoint, the wallet application performs a WebFinger query that looks for the PayID Discovery service provider, like this:
```

Comment:
>  Alice logs in to the wallet provider and enter Bob's PayID

Should that be "enters Bob's PAyID"?

Reply:
> If the wallet application is set up to use the PayID Discovery endpoint

In the interest of simplicity, maybe we could just say that the wallet _is_ set up to use discovery? That would get rid of a few words, too.

Reply:
Not in scope for this PR but a diagram would really help illustrate this flow!